unfetter-discover/unfetter-ui
# Issue 1306/remove beta

Fixes unfetter-discover/unfetter#1306

Removed beta from infobars on summary and wizard screens, and updated logo for Baselines.

### To test:
1. Pull the branch
2. Restart UI
3. Access Unfetter and click navigation dropdown - icon for baselines should look like this:

![icon-baselines](https://user-images.githubusercontent.com/34275439/43472523-a48e7748-94bb-11e8-9b06-15fcf8a55463.png)

4. Test navigation into baselines, as well as into and out of baselines wizard
